### DISCUSSION TITLE
Potentially incorrect display resolution reported in new 2024 Miyoo Mini firmware/board rev.

### DISCUSSION THREAD

**Discussion by diviocity** [OnionUI/Onion#1673] [Support] [answered]
Hi, I recently picked up a Miyoo Mini V4 and discovered it had an updated board revision and firmware. One of the changes that came with this revision is the addition of a real time clock (RTC) which I posted about on Reddit [here](https://www.reddit.com/r/MiyooMini/comments/1fo3140/miyoo_mini_v4_seems_to_have_rtc_now/).

However, thanks to another user, I also noticed the screen resolution being reported by emulators was 640x480. You can see that user's report [here](https://www.reddit.com/r/MiyooMini/comments/1foa1vo/onionos_reporting_mini_v4_screen_as_640x480/). He has the same 2024 firmware as I do.

What I'm unsure about is if our displays are the 560p screens that have been shipping in these devices or if they're 480p like the older ones. I don't think I can report this as an issue until I can figure that out. I took a system log snapshot via the tweaks app and will attach those here, in case they can answer my question. I saw in dmesg.log that FB_DEVICE reports 752x560, so my best guess would be that it's actually a 752x560 screen... but I'm not 100% sure what I'm looking at and would like some way to confirm that it actually is that resolution before moving onto saying this is an OnionOS issue.

Any help would be appreciated!
[log_export.zip](https://github.com/user-attachments/files/17120974/log_export.zip)

Latest referenced firmware: 2023 10 27 1401
New firmware on new minis: 2024 07 21 1632

**Comment by diviocity** [comment, marked as answer]
Confirmed the screen is 752x560 still and figured out why scaling is broken on the new firmware. Looks like runtime.sh is only changing the resolution if the firmware version = 202310271401, so we can just change that to fix things again until the Onion team releases a proper update:

1. Mount SD card to PC
2. Find "runtime.sh" under /.tmp_update/
3. Right click, edit with Notepad
4. Edit -> Find, look for "miyoo_version=202310271401"
5. replace "202310271401" with "202407211632"
6. Save, unmount
7. Slot into MM and enjoy!

**Reply by purelygrey** [reply]
Thank you very much for the workaround! You saved me from a headache

**Reply by TripleAkat** [reply]
Same
